frontend/test-results/e2e-test-Executor-UI-Tests-640df--a-todo-and-start-execution/error-context.md

```
@@ -0,0 +1,108 @@
+- Following Playwright test failed.
+- Explain why, be concise, respect Playwright best practices.
+- Provide a snippet of code with the fix, if possible.
+
+# Test info
+
+- Name: e2e-test.spec.ts >> Executor UI Tests >> should create a todo and start execution
+- Location: e2e-test.spec.ts:16:3
+
+# Error details
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)